reporting/impl/src/main/resources/reports/resources/css/tech-report-punchcard.css

```diff
@@ -66,30 +62,21 @@ table.technologiesPunchCard tbody tr td, table.technologiesPunchCard thead tr td
     max-width: 47px;
 }
 
-table.technologiesPunchCard tbody tr td:nth-last-child(2),
 table.technologiesPunchCard tbody tr td:nth-last-child(3),
-table.technologiesPunchCard tbody tr td:nth-last-child(4),
-table.technologiesPunchCard thead tr td:nth-last-child(2),
-table.technologiesPunchCard thead tr td:nth-last-child(3),
-table.technologiesPunchCard thead tr td:nth-last-child(4) {
-    width: 57px;
-    min-width: 57px;
-    max-width: 57px;
-}
+table.technologiesPunchCard thead tr td:nth-last-child(3) { width: 70px; min-width: 70px; max-width: 70px; }
+
+table.technologiesPunchCard tbody tr td:nth-last-child(2),
+table.technologiesPunchCard thead tr td:nth-last-child(2) { width: 57px; min-width: 57px; max-width: 57px; }
```

Comment:
Hi!

Can you also make sure that ...

table.technologiesPunchCard thead tr td:nth-last-child(3) { width: 70px; min-width: 70px; max-width: 70px; }

... is kept?

Probably you will need to update it to

table.technologiesPunchCard thead tr td:nth-last-child(4) { width: 70px; min-width: 70px; max-width: 70px; }

Reply:
Ok, so talking in terms of requirements, your request is to have wider columns in `Stats` section, right? So that numbers have more space to be displayed. 
Having only more width for table cell (setting their width to `70px`), like in the screenshot above, will cause a misalignment between labels and values just like in the next screenshot.
![screenshot from 2018-05-29 12-39-43](https://user-images.githubusercontent.com/7288588/40654398-89e7ab70-633e-11e8-9198-e6be88bd9740.png)

We can work on inner `<div>` to better place the text to obtain something like this:
![screenshot from 2018-05-29 12-40-44](https://user-images.githubusercontent.com/7288588/40654471-b7c3a3dc-633e-11e8-8a61-ba2798b9c279.png)
where both `Size (MB)` and `Libraries` labels are aligned with the data below them.
@jsight i use an in-line style for div `width: inherit;transform: rotate(180deg) translateX(30%);`

Reply:
The wider column was only for ":nth-last-child(3)" -> the "Size (MB)" column.

Reply:
Ok, great, thanks @Maarc: only `Size (MB)` column wider :+1: